### DISCUSSION TITLE
It is not clear that the list is private or public

### DISCUSSION THREAD

**Discussion by haxpor** [community/community#8770] [Other Feature Feedback, Questions, & Ideas]
According to the title, I have to go on with incognito browser session to check whether other people will be able to see my saved lists.
It turns out to be private which is what I want.

It would be better in my humble opinion to clearly state that the list is private.
Thanks!

**Comment by brianlovin** [comment]
Lists are public by default, sorry for the confusion here @haxpor. We have rolled out some changes that will ensure Lists appear to signed-out users as well.
